webdriver manager adds IEDriver for x64 but tries to launch x86

I wanted to add IEDriver via webdriver manager to my system. With `webdriver-manager update --ie` with a Win10 x64 system, the webdriver fetches the x64 driver. 
```
[18:07:42] I/downloader - IEDriverServer: downloading version _x64_2.53.1
[18:07:42] I/downloader - curl -o C:\Users\XXXX\AppData\Roaming\npm\node_modules\protractor\node_modules\webdriver-manager\selenium/IEDriverServer_x64_2.53.1.zip https://selenium-release.storage.googleapis.com/2.53/IEDriverServer_x64_2.53.1.zip
```


Then, on the hub, I try to launch a session with IE it says that the x86 driver is not found

```
System info: host: 'XXXX-PC', ip: 'XX.XXX.XXX.XX', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_101'
Driver info: driver.version: unknown
        at org.openqa.selenium.remote.server.DefaultDriverProvider.callConstructor(DefaultDriverProvider.java:113)
        at org.openqa.selenium.remote.server.DefaultDriverProvider.newInstance(DefaultDriverProvider.java:97)
        at org.openqa.selenium.remote.server.DefaultDriverFactory.newInstance(DefaultDriverFactory.java:60)
        at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(DefaultSession.java:222)
        at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(DefaultSession.java:1)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultSession$1.run(DefaultSession.java:176)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultDriverProvider.callConstructor(DefaultDriverProvider.java:103)
        ... 9 more
Caused by: java.lang.IllegalStateException: The driver executable does not exist: C:\Users\Tiago\AppData\Roaming\npm\node_modules\protractor\node_modules\webdriver-manager\selenium\IEDriverServer_Win32_2.53.1.exe
        at com.google.common.base.Preconditions.checkState(Preconditions.java:199)
        at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:121)
        at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:116)
        at org.openqa.selenium.ie.InternetExplorerDriverService.access$0(InternetExplorerDriverService.java:1)
        at org.openqa.selenium.ie.InternetExplorerDriverService$Builder.findDefaultExecutable(InternetExplorerDriverService.java:167)
        at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:296)
        at org.openqa.selenium.ie.InternetExplorerDriver.setupService(InternetExplorerDriver.java:251)
        at org.openqa.selenium.ie.InternetExplorerDriver.<init>(InternetExplorerDriver.java:172)
        at org.openqa.selenium.ie.InternetExplorerDriver.<init>(InternetExplorerDriver.java:150)
        ... 14 more
18:06:15.098 WARN - Exception: The driver executable does not exist: C:\Users\XXXX\AppData\Roaming\npm\node_modules\protractor\node_modules\webdriver-manager\selenium\IEDriverServer_Win32_2.53.1.exe
```

--- a/lib/cmds/start.ts
+++ b/lib/cmds/start.ts
@@ -130,7 +130,6 @@ function start(options: Options) {
     // https://bugs.openjdk.java.net/browse/JDK-6202721
     args.push('-Djava.security.egd=file:///dev/./urandom');
   }
-
   if (options[Opt.LOGGING].getString()) {
     if (path.isAbsolute(options[Opt.LOGGING].getString())) {
       loggingFile = options[Opt.LOGGING].getString();
@@ -154,14 +153,14 @@ function start(options: Options) {
         path.resolve(outputDir, binaries[GeckoDriver.id].executableFilename(osType)));
   }
   if (downloadedBinaries[IEDriver.id] != null) {
-    if (options[Opt.IE32]) {
+    if (options[Opt.IE32].getBoolean()) {
       binaries[IEDriver.id].arch = 'Win32';
     }
     args.push(
         '-Dwebdriver.ie.driver=' +
         path.resolve(outputDir, binaries[IEDriver.id].executableFilename(osType)));
   }
-  if (options[Opt.EDGE]) {
+  if (options[Opt.EDGE] && options[Opt.EDGE].getString()) {
     // validate that the file exists prior to adding it to args
     try {
       let edgeFile = options[Opt.EDGE].getString();

